yang/example-scheduled-backup.yang

```diff
@@ -0,0 +1,77 @@
+module example-scheduled-backup-module {
+  yang-version 1.1;
+  namespace "http://example.com/example-scheduled-backup-module";
+  prefix "exscback";
+
+  import ietf-inet-types {
+    prefix "inet";
+  }
+
+  import ietf-schedule {
+    prefix "schedule";
+  }
+
+  organization
+    "Example, Inc.";
+
+  contact
+    "Support at example.com";
+
+  description
+    "Example of a module defining an scheduled based backup operation.";
+
+  revision "2023-01-19" {
+    description
+      "Initial Version.";
+    reference
+      "RFC XXXX: A YANG Data Model for Scheduling.";
+    }
+
+  container scheduled-backup-tasks {
+    description
+      "A container for backing up all current running configuration
+       on the device.";
+
+    choice local-or-remote {
+      description
+        "Specify whether the configuration to be backed up is local or
+         remote.";
+      case local {
+        description
+          "Configuration parameter for backing up of local device.";
```

Comment:
```suggestion
          "Configuration parameters for backing up of local device.";
```

Reply:
Plural form? We only defined one parameter under this case.

Reply:
We need to distinguish the intent of the container vs the current parameter'(s) that are listed out there.